lib/teleterm/daemon/daemon.go

```diff
@@ -133,15 +134,13 @@ func (s *Service) CreateGateway(ctx context.Context, params CreateGatewayParams)
 	return gateway, nil
 }
 
+type GatewayCreator interface {
```

Comment:
nit: I can't tell if it's nicer to keep interfaces like this defined closer to the config ? otoh, here it's closer to the actual usage.

Reply:
> nit: I can't tell if it's nicer to keep interfaces like this defined closer to the config ? otoh, here it's closer to the actual usage.

Sorry, forgot to leave the comment, I decided to keep it closer to the actual usage.